src/App.jsx

```diff
@@ -13,14 +14,15 @@ function App() {
 }
 const [getUser, setGetUser] =useState(noUserSelected)
 
+const [articleId, setArticleId] = useState([])
```

Comment:
not sure why you have a state for articleId when both components you are passing it to will access to it either from the data coming back or useParams()